book/src/applet/prelude/button1.rs

```diff
@@ -30,15 +30,15 @@ fn main() {
     //{ ANCHOR: loop
     // For each button on the board.
     for index in 0 .. count {
-    //} ANCHOR_END: loop
+        //} ANCHOR_END: loop
```

Comment:
The idea was to keep the ANCHOR_END aligned with its ANCHOR, even though from a pure "Rust comment" perspective this is wrong. Those comments are actually outside Rust and happens before. They are just comments for the file to still have a Rust meaning.

Reply:
Removed, I think my formatter did this.